### PR TITLE
feat: add events and CFPs, update others

### PR DESCRIPTION
**Pull request by Lenormju**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -188,7 +188,8 @@ All the data (past and coming) is available publicly in JSON:
 * 19-21: [GoLab](https://golab.io/) - Florence (Italy) <a href="https://www.papercall.io/golab2023"><img alt="CFP GoLab 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2021-May-2023&color=red"></a>
 * 19-21: [RustLab](https://rustlab.it/) - Florence (Italy) <a href="https://www.papercall.io/rustlab2023"><img alt="CFP GoLab 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2026-May-2023&color=red"></a>
 * 21-24: [JCON WORLD](https://2023.world.jcon.one/) - Online <a href="https://sessionize.com/jcon-world-2023/"><img alt="CFP JCon World 2023" src="https://img.shields.io/static/v1?label=CFP&message=until%2030-June-2023&color=red"></a>
-* 23: [DevOps D-Day #8](https://www.devopsdday.com/) - Marseille (France) <a href="https://hopscotch.key4events.com/abstract.aspx?e=321&c=2683"><img alt="CFP DevOps D-Day #8" src="https://img.shields.io/static/v1?label=CFP&message=until%2014-July-2023&color=green"></a>
```
Comment (Lenormju) [inline README.md:191]: **note**: updated CPF status

File: README.md [hunk 2]
```diff
@@ -214,7 +215,7 @@ All the data (past and coming) is available publicly in JSON:
 ### January
 
 * 20: [Voxxed Days Ticino](https://voxxeddays.com/ticino/) - Lugano (Switzerland)
-* 31-03/02: [SnowCamp](https://snowcamp.io/fr/) - Grenoble (France)
+* 31-03/02: [SnowCamp](https://snowcamp.io/) - Grenoble (France)<a href="https://snowcamp.io/"><img alt="CFP SnowCamp" src="https://img.shields.io/static/v1?label=CFP&message=until%2015-January-2024&color=green"></a>
```
Comment (Lenormju) [inline README.md:218]: **question(non-blocking)**: they also provide a *springboard* ("tremplin") to coach potential speakers, should it be added too ?

[the remark this answers is not in the document]
Comment (scraly) [reply README.md:218]: please add a space after "(France)" :)

Comment (scraly) [reply README.md:218]: The "tremplin" is only a concept in France for the moment.
The link is available in the snowcamp website?

**Review by scraly — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Lenormju) [reply README.md:218]: Link for the tremplin : https://conference-hall.io/public/event/7tzYnx2FMO3J2q4N7JiE
It does not seem to be present on their website, but they shared it on their official LinkedIn : https://www.linkedin.com/feed/update/urn:li:activity:7088100326846734337/

**Comment by scraly** [comment]
thanks

**Pull request merged**